pkgs/by-name/te/television/wrapper.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildEnv,
+  television,
+  targetPackages,
+  makeBinaryWrapper,
+}:
+
+lib.makeOverridable (
+  pkgs: f:
+  buildEnv {
+    inherit (lib.appendToName "with-pkgs" television) name;
+    inherit (television) meta;
+
+    paths = f pkgs;
+
+    pathsToLink = [ "/bin" ];
+
+    nativeBuildInputs = [ makeBinaryWrapper ];
+
+    postBuild = ''
+      mv "$out/bin" "$out/share"
+
+      mkdir -p "$out/bin"
+      makeWrapper ${lib.getExe television} $out/bin/tv \
+        --prefix PATH : "$out/share"
```

```suggestion
      wrapProgram $out/bin/tv \
        --prefix PATH : "${lib.makeBinaryPath (f pkgs)}"
```
And we can simplify this whole step a fair bit

---

Good idea.